config.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	homedir "github.com/mitchellh/go-homedir"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	rootConfigPathName     = ".fn"
+	contextsPathName       = "contexts"
+	configName             = "config"
+	contextConfigFileName  = "config.yaml"
```

Comment:
Is it one file per context or all the contexts in this one file?  And if all in one, should default also be in that same file?

Reply:
Its one file per context with `default.yaml` being created initially.